.travis.yml

```diff
@@ -8,7 +8,7 @@ stages:
 jobs:
   include:
     - stage: test
-      script: travis_wait 14 ./gradlew build
+      script: travis_wait 21 ./gradlew build
```

Comment:
Extending the dev loop by 7 minutes is a lot.
`virtual-users` have a Docker Jira with 100k users, which starts in 7 minutes (with 1 minute upgrade task time and no jira-home/DB parallelization). `JiraCoreFormula` might be a bit too slow. We might leverage some form of caching (e.g. the emergent dataset).
The dev loop penalty is induced by the inefficiencies in `docker-infrastructure`, so I won't require this to be fixed here. However, if we do expand the coverage over multiple Jira versions, I'd ask to parallelize the test internally to keep the dev loop penalty reasonably low.